app/javascript/hyacinth_ui_v1/components/dynamic_fields/DynamicFieldForm.js

```diff
@@ -61,14 +61,14 @@ class DynamicFieldForm extends React.Component {
       }));
     }
 
-    hyacinthApi.get('vocabularies')
-      .then((res) => {
-        const { vocabularies } = res.data;
-
-        this.setState(produce((draft) => {
-          draft.vocabularies = vocabularies.map(v => ({ value: v.stringKey, label: v.label }));
-        }));
-      });
+    // hyacinthApi.get('vocabularies')
```

Comment:
these comment-out blocks are to be restored for `dynamic fields controlled vocabulary selection` subsequently, right?

Reply:
yes, i just didn't want them to introduce errors on the UI